components/brave_shields/browser/ad_block_service.cc

```diff
@@ -142,8 +142,6 @@ void AdBlockPrefService::OnPreferenceChanged(const std::string& pref_name) {
   }
   bool enabled = prefs_->GetBoolean(pref_name);
   g_brave_browser_process->ad_block_service()->EnableTag(tag, enabled);
-  g_brave_browser_process->ad_block_regional_service_manager()->EnableTag(
-      tag, enabled);
```

Comment:
@jumde I think you may have touched the wrong lines here

Reply:
@antonok-edm - fixed